_posts/2016-04-09-linux-mint-razer-blade.markdown

```diff
@@ -0,0 +1,97 @@
+
+I've done 3 or 4 full builds of Linux before, all with Mint. One on a Chromebook, but mostly on machines with slightly dated hardware. This build has by far been the hardest, and I learned the hard way that Linux isn't as stable on "new" hardware.
+
+###Chapter 1: Pre-install
+**Expand SSD storage (optional)**
+
+The SSD in the Full HD edition is about 250 GB. If you're dual-booting, like I did, this doesn't leave much space to split. If you're going to replace the ssd with a larger one, now's the time to do it. Heres a video on how:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/xjM-ByOE0fQ" frameborder="0" allowfullscreen></iframe>
+
+**Prepare your SSD (optional)**
+
+Theres loads of stuff that can be done to reduce the wear on a SSD. Several things on [this list](https://sites.google.com/site/easylinuxtipsproject/ssd) can, and should, only be done before install. Its something to consider.
+
+**Get bootable USB drive**
+
+Get the download for Mint from the [downloads page](https://www.linuxmint.com/download.php), I **highly recommend** you choose the Mate version over Cinnamon. This will come into play later. Although, if you choose Cinnamon and run into problems, there're ways to switch after the fact.
+
+Open up the Linux Mint [User Guide](https://www.linuxmint.com/documentation.php) and follow the instructions to verify your MD5. You'll also want this open for details on most of the initial setup process. This post only covers steps beyond the guide.
+
+Use [this tutorial](https://community.linuxmint.com/tutorial/view/744) to mount the downloaded `.iso` to a USB drive thats at least 2GB.
+
+###Chapter 2: Install
+Plug the Razer Blade into a power source. Follow the directions in the user guide to boot from USB and start the install process.
+
+If you end up on the partition menu to customize the size of the install, make sure to add a few GB partition as "swap area". Among other things, this is important for resuming from a suspend. Learned this the hard way.
+
+Follow the rest of the directions to install completely and remove the USB drive (it'll prompt you).
+
+After install, boot up the machine, sign in, and install any updates from the Update Manager.
+
+###Chapter 3: Fixing hardware issues
+There are several parts of the hardware that just aren't hooked up right with the base install. I ran into issues with suspend black-screening indefinitely, two-finger scroll not working and other touchpad issues, and the internal webcam not working.
+
+The only concrete and consistent fix I've found is to change the version of the Linux kernel thats being used. This is a critical part of the system, so use caution, but I was able to fix most of my hardware issues this way.
```

Comment:
Didn't use the word "update" cus it suggests that its a change to a stable version. Its a change to a more recent version, but I won't call it an "update"